### PR TITLE
8.1.1

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- Automated pull request created via HAL 9000.  See changes below for details.

# Review By (Date)
- As soon as possible

The following changes since commit 36152f17cd5a0b7cea923d958b9a0c20af583bd1:

  Merge pull request #60 from SU-SOE/release-8.1.1 (2020-08-12 13:03:47 -0600)

are available in the Git repository at:

  ./ 

for you to fetch changes up to bee1b333dbb554c391567786dc81ab67bebf60ab:

  8.1.1 (2020-09-16 11:21:48 -0600)

----------------------------------------------------------------
Ian Monroe (1):
      8.1.1

Sherakama (4):
      Update composer.json (#63)
      D8CORE-1609: Sync super footer/global messages from stanford_profile (#64)
      D8CORE-000: Lockup tests and shuffle dance (#67)
      Update LocalFooterCest.php (#68)

sws-developers@lists.stanford.edu (2):
      Back to dev
      Back to dev

 CHANGELOG.md                                       |   9 +
 composer.json                                      |  43 ++-
 config/sync/block.block.config_pages.yml           |   2 +-
 ...ock.block.soe_basic_config_pages_global_msg.yml |  29 ++
 ....block.soe_basic_config_pages_local_footer.yml} |   4 +-
 ...k.block.soe_basic_config_pages_super_footer.yml |  29 ++
 ...ford_basic_config_pages_stanford_global_msg.yml |  29 ++
 ...rd_basic_config_pages_stanford_super_footer.yml |  29 ++
 config/sync/config_ignore.settings.yml             |   3 +-
 .../config_pages.type.stanford_global_message.yml  |  17 +
 .../config_pages.type.stanford_super_footer.yml    |  17 +
 ...onfig_pages.stanford_global_message.default.yml |  67 ++++
 ....config_pages.stanford_local_footer.default.yml | 155 +++++++-
 ....config_pages.stanford_super_footer.default.yml |  82 +++++
 ...ntity_form_display.media.embeddable.default.yml |  44 +++
 ...form_display.media.embeddable.media_library.yml |  43 +++
 ...ity_form_display.node.stanford_page.default.yml |   5 +-
 ...onfig_pages.stanford_global_message.default.yml |  81 +++++
 ....config_pages.stanford_local_footer.default.yml |  74 +++-
 ....config_pages.stanford_super_footer.default.yml |  93 +++++
 ...ntity_view_display.media.embeddable.default.yml |  38 ++
 ...view_display.media.embeddable.media_library.yml |  39 ++
 ...aragraph_row.node_stanford_page_row.default.yml |  25 ++
 config/sync/core.extension.yml                     |   1 +
 .../sync/core.menu.static_menu_link_overrides.yml  |  86 ++++-
 config/sync/extlink.settings.yml                   |  26 ++
 ...anford_global_message.su_global_msg_enabled.yml |  23 ++
 ...tanford_global_message.su_global_msg_header.yml |  19 +
 ...stanford_global_message.su_global_msg_label.yml |  19 +
 ....stanford_global_message.su_global_msg_link.yml |  23 ++
 ...anford_global_message.su_global_msg_message.yml |  27 ++
 ....stanford_global_message.su_global_msg_type.yml |  23 ++
 ....stanford_local_footer.su_local_foot_line_1.yml |  19 +
 ....stanford_local_footer.su_local_foot_line_2.yml |  19 +
 ....stanford_local_footer.su_local_foot_line_3.yml |  19 +
 ....stanford_local_footer.su_local_foot_line_4.yml |  19 +
 ....stanford_local_footer.su_local_foot_line_5.yml |  19 +
 ...stanford_local_footer.su_local_foot_loc_img.yml |  38 ++
 ...tanford_local_footer.su_local_foot_loc_link.yml |  23 ++
 ....stanford_local_footer.su_local_foot_loc_op.yml |  23 ++
 ...s.stanford_local_footer.su_local_foot_pr_co.yml |  27 ++
 ...s.stanford_local_footer.su_local_foot_se_co.yml |  27 ++
 ...s.stanford_local_footer.su_local_foot_tr_co.yml |  27 ++
 ...stanford_local_footer.su_local_foot_use_loc.yml |  23 ++
 ...tanford_local_footer.su_local_foot_use_logo.yml |  23 ++
 ...stanford_super_footer.su_super_foot_enabled.yml |  23 ++
 ...tanford_super_footer.su_super_foot_intranet.yml |  23 ++
 ...es.stanford_super_footer.su_super_foot_link.yml |  23 ++
 ...es.stanford_super_footer.su_super_foot_text.yml |  27 ++
 ...s.stanford_super_footer.su_super_foot_title.yml |  19 +
 ...edia.embeddable.field_media_embeddable_code.yml |  19 +
 ...ia.embeddable.field_media_embeddable_oembed.yml |  19 +
 ...field.node.stanford_page.su_page_components.yml |  12 +-
 ...w.node_stanford_page_row.su_page_components.yml |  60 ++++
 ....storage.config_pages.su_global_msg_enabled.yml |  22 ++
 ...d.storage.config_pages.su_global_msg_header.yml |  25 ++
 ...ld.storage.config_pages.su_global_msg_label.yml |  25 ++
 ...eld.storage.config_pages.su_global_msg_link.yml |  23 ++
 ....storage.config_pages.su_global_msg_message.yml |  23 ++
 ...eld.storage.config_pages.su_global_msg_type.yml |  40 +++
 ...d.storage.config_pages.su_local_foot_line_1.yml |  25 ++
 ...d.storage.config_pages.su_local_foot_line_2.yml |  25 ++
 ...d.storage.config_pages.su_local_foot_line_3.yml |  25 ++
 ...d.storage.config_pages.su_local_foot_line_4.yml |  25 ++
 ...d.storage.config_pages.su_local_foot_line_5.yml |  25 ++
 ....storage.config_pages.su_local_foot_loc_img.yml |  34 ++
 ...storage.config_pages.su_local_foot_loc_link.yml |  23 ++
 ...d.storage.config_pages.su_local_foot_loc_op.yml |  64 ++++
 ...ld.storage.config_pages.su_local_foot_pr_co.yml |  23 ++
 ...ld.storage.config_pages.su_local_foot_se_co.yml |  23 ++
 ...ld.storage.config_pages.su_local_foot_tr_co.yml |  23 ++
 ....storage.config_pages.su_local_foot_use_loc.yml |  22 ++
 ...storage.config_pages.su_local_foot_use_logo.yml |  22 ++
 ...ield.storage.config_pages.su_lockup_options.yml |   3 +
 ....storage.config_pages.su_super_foot_enabled.yml |  22 ++
 ...storage.config_pages.su_super_foot_intranet.yml |  23 ++
 ...eld.storage.config_pages.su_super_foot_link.yml |  23 ++
 ...eld.storage.config_pages.su_super_foot_text.yml |  23 ++
 ...ld.storage.config_pages.su_super_foot_title.yml |  25 ++
 ...d.storage.media.field_media_embeddable_code.yml |  23 ++
 ...storage.media.field_media_embeddable_oembed.yml |  22 ++
 .../sync/field.storage.node.su_page_components.yml |  10 +-
 .../field.storage.node.su_person_first_name.yml    |   4 +
 .../field.storage.node.su_person_full_title.yml    |   4 +
 .../field.storage.node.su_person_last_name.yml     |   4 +
 config/sync/field.storage.node.su_person_photo.yml |   4 +
 ...ld.storage.paragraph_row.su_page_components.yml |  21 ++
 config/sync/filter.format.stanford_html.yml        |   2 +-
 ...out_library.layout.stanford_basic_page_full.yml |   2 +-
 config/sync/media.type.embeddable.yml              |  40 +++
 ....paragraphs_row_type.node_stanford_page_row.yml |   6 +
 config/sync/rest.resource.entity.paragraph_row.yml |  18 +
 ...axonomy_menu.taxonomy_menu.event_types_menu.yml |   2 +-
 ...axonomy_menu.taxonomy_menu.person_type_menu.yml |   2 +-
 ...omy_menu.taxonomy_menu.topics_taxonomy_menu.yml |   2 +-
 config/sync/user.role.anonymous.yml                |   2 +
 config/sync/user.role.authenticated.yml            |   2 +
 config/sync/user.role.site_builder.yml             |  13 +
 config/sync/user.role.site_developer.yml           |  13 +
 config/sync/user.role.site_editor.yml              |   5 +
 config/sync/user.role.site_manager.yml             |   7 +
 ...views.view.stanford_person_list_terms_first.yml |  16 +-
 .../node/4b8018dc-49a6-4018-9c54-e8c3e462beee.json |  32 +-
 .../node/6d51339d-ff67-498d-98e9-d8228d36fd51.json |  32 +-
 .../node/72f0069b-f1ec-4122-af73-6aa841faea90.json | 106 +++---
 .../node/9153cfe9-e078-4048-b5c6-1282815a73e4.json | 196 ----------
 .../node/bc484600-a21a-4497-b9f7-f928474b511c.json |  32 +-
 .../node/d21d84f0-22e7-4596-82cb-9209841a3276.json |  32 +-
 .../node/e18a741f-7d86-492a-a02f-7ca73989ca13.json |  32 +-
 .../35b55d09-f3e6-4134-87c1-4a8ea132940f.json      |  23 +-
 .../79c98714-281a-450d-b574-2971426b1f3e.json      |   6 +-
 .../8f7b2d0f-f883-43c8-9782-5e8b88fd98dc.json      |  15 +-
 .../9954cc81-919b-4498-9151-bf930831fca7.json      |  22 +-
 .../9df33d0d-239e-4b51-9d97-8703187629b4.json      |  11 +-
 .../b7abb133-a21b-4a1a-a13c-f630aad2ce7e.json      |  19 +-
 .../bf52b16d-03ef-41dc-b906-4a3a88c2e9ce.json      |  13 +-
 .../c77c8f62-721c-475d-bd5f-ba5ddce96836.json      |  15 +-
 .../cffc69b9-51a3-4e0e-a1b3-864202e42e6c.json      |  19 +-
 .../d800e9c3-5a54-4241-983e-87c42f31b557.json      |  13 +-
 .../fbbaaecc-80fc-494a-8bce-83638d1d668d.json      |  21 +-
 .../0427d80b-1f0a-43e1-b902-03d7df3d8523.json      |  93 +++++
 .../27047c34-f161-45d4-ab65-c7d7c0e98765.json      |  93 +++++
 .../4a607606-12f6-482c-8cd7-a57d64aeff6b.json      |  93 +++++
 .../71b78295-dfd1-4eb3-8d20-2013d5b64f69.json      |  93 +++++
 .../8f739f14-a899-4880-9c11-5929f0d4ddde.json      |  93 +++++
 .../95514879-c3d0-44cc-a8bd-987f547fa1a8.json      |  93 +++++
 .../a9a54e9d-e433-408f-be3d-141d3cb3aad6.json      |  93 +++++
 .../b9027856-bb49-4c3b-9367-87a4e3731421.json      |  93 +++++
 .../ec2d0801-e418-4bec-a1f9-064c349148d7.json      |  93 +++++
 .../ed135145-45ed-4955-a1b6-1141bc869568.json      |  93 +++++
 soe_profile.info.yml                               |  57 ++-
 soe_profile.post_update.php                        | 115 ++++++
 .../acceptance/Contrib/NodeRevisionDeleteCest.php  |   8 +-
 .../acceptance/GlobalMessage/GlobalMessageCest.php |  97 +++++
 .../{ => LocalFooter}/LocalFooterCest.php          |  27 ++
 .../LocalFooter/LocalFooterLockupCest.php          | 399 +++++++++++++++++++++
 tests/codeception/acceptance/LocalFooter/logo.jpg  | Bin 0 -> 35675 bytes
 .../LockupSettings/LockupSettingsCest.php          |  27 +-
 tests/codeception/acceptance/Media/MediaCest.php   |  91 +++++
 .../acceptance/Media/MediaPermissionsCest.php      |  52 +++
 tests/codeception/acceptance/MediaCest.php         |  17 -
 .../acceptance/Paragraphs/BannerCest.php           |  18 +-
 .../acceptance/Paragraphs/MediaWithCaptionCest.php |  18 +-
 .../acceptance/Paragraphs/StanfordCTAListCest.php  |  19 +-
 .../acceptance/Paragraphs/StanfordCardCest.php     |  18 +-
 .../acceptance/Paragraphs/StanfordImageCTACest.php |  19 +-
 .../acceptance/Paragraphs/StanfordStoriesCest.php  |  19 +-
 .../acceptance/SuperFooter/SuperFooterCest.php     |  93 +++++
 .../BasicPage/BasicPageParagraphsCest.php          |  15 +-
 .../codeception/functional/Contrib/ExtLinkCest.php |  52 +++
 .../functional/Paragraphs/WYSIWYGCest.php          |  32 +-
 151 files changed, 4815 insertions(+), 542 deletions(-)
 create mode 100644 config/sync/block.block.soe_basic_config_pages_global_msg.yml
 rename config/sync/{block.block.config_pages_2.yml => block.block.soe_basic_config_pages_local_footer.yml} (92%)
 create mode 100644 config/sync/block.block.soe_basic_config_pages_super_footer.yml
 create mode 100644 config/sync/block.block.stanford_basic_config_pages_stanford_global_msg.yml
 create mode 100644 config/sync/block.block.stanford_basic_config_pages_stanford_super_footer.yml
 create mode 100644 config/sync/config_pages.type.stanford_global_message.yml
 create mode 100644 config/sync/config_pages.type.stanford_super_footer.yml
 create mode 100644 config/sync/core.entity_form_display.config_pages.stanford_global_message.default.yml
 create mode 100644 config/sync/core.entity_form_display.config_pages.stanford_super_footer.default.yml
 create mode 100644 config/sync/core.entity_form_display.media.embeddable.default.yml
 create mode 100644 config/sync/core.entity_form_display.media.embeddable.media_library.yml
 create mode 100644 config/sync/core.entity_view_display.config_pages.stanford_global_message.default.yml
 create mode 100644 config/sync/core.entity_view_display.config_pages.stanford_super_footer.default.yml
 create mode 100644 config/sync/core.entity_view_display.media.embeddable.default.yml
 create mode 100644 config/sync/core.entity_view_display.media.embeddable.media_library.yml
 create mode 100644 config/sync/core.entity_view_display.paragraph_row.node_stanford_page_row.default.yml
 create mode 100644 config/sync/extlink.settings.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_enabled.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_header.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_label.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_link.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_message.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_global_message.su_global_msg_type.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_line_1.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_line_2.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_line_3.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_line_4.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_line_5.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_loc_img.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_loc_link.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_loc_op.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_pr_co.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_se_co.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_tr_co.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_use_loc.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_use_logo.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_super_footer.su_super_foot_enabled.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_super_footer.su_super_foot_intranet.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_super_footer.su_super_foot_link.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_super_footer.su_super_foot_text.yml
 create mode 100644 config/sync/field.field.config_pages.stanford_super_footer.su_super_foot_title.yml
 create mode 100644 config/sync/field.field.media.embeddable.field_media_embeddable_code.yml
 create mode 100644 config/sync/field.field.media.embeddable.field_media_embeddable_oembed.yml
 create mode 100644 config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_enabled.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_header.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_label.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_link.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_message.yml
 create mode 100644 config/sync/field.storage.config_pages.su_global_msg_type.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_line_1.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_line_2.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_line_3.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_line_4.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_line_5.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_loc_img.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_loc_link.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_loc_op.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_pr_co.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_se_co.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_tr_co.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_use_loc.yml
 create mode 100644 config/sync/field.storage.config_pages.su_local_foot_use_logo.yml
 create mode 100644 config/sync/field.storage.config_pages.su_super_foot_enabled.yml
 create mode 100644 config/sync/field.storage.config_pages.su_super_foot_intranet.yml
 create mode 100644 config/sync/field.storage.config_pages.su_super_foot_link.yml
 create mode 100644 config/sync/field.storage.config_pages.su_super_foot_text.yml
 create mode 100644 config/sync/field.storage.config_pages.su_super_foot_title.yml
 create mode 100644 config/sync/field.storage.media.field_media_embeddable_code.yml
 create mode 100644 config/sync/field.storage.media.field_media_embeddable_oembed.yml
 create mode 100644 config/sync/field.storage.paragraph_row.su_page_components.yml
 create mode 100644 config/sync/media.type.embeddable.yml
 create mode 100644 config/sync/react_paragraphs.paragraphs_row_type.node_stanford_page_row.yml
 create mode 100644 config/sync/rest.resource.entity.paragraph_row.yml
 delete mode 100644 content/node/9153cfe9-e078-4048-b5c6-1282815a73e4.json
 create mode 100644 content/paragraph_row/0427d80b-1f0a-43e1-b902-03d7df3d8523.json
 create mode 100644 content/paragraph_row/27047c34-f161-45d4-ab65-c7d7c0e98765.json
 create mode 100644 content/paragraph_row/4a607606-12f6-482c-8cd7-a57d64aeff6b.json
 create mode 100644 content/paragraph_row/71b78295-dfd1-4eb3-8d20-2013d5b64f69.json
 create mode 100644 content/paragraph_row/8f739f14-a899-4880-9c11-5929f0d4ddde.json
 create mode 100644 content/paragraph_row/95514879-c3d0-44cc-a8bd-987f547fa1a8.json
 create mode 100644 content/paragraph_row/a9a54e9d-e433-408f-be3d-141d3cb3aad6.json
 create mode 100644 content/paragraph_row/b9027856-bb49-4c3b-9367-87a4e3731421.json
 create mode 100644 content/paragraph_row/ec2d0801-e418-4bec-a1f9-064c349148d7.json
 create mode 100644 content/paragraph_row/ed135145-45ed-4955-a1b6-1141bc869568.json
 create mode 100644 soe_profile.post_update.php
 create mode 100644 tests/codeception/acceptance/GlobalMessage/GlobalMessageCest.php
 rename tests/codeception/acceptance/{ => LocalFooter}/LocalFooterCest.php (75%)
 mode change 100755 => 100644
 create mode 100644 tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php
 create mode 100644 tests/codeception/acceptance/LocalFooter/logo.jpg
 create mode 100644 tests/codeception/acceptance/Media/MediaCest.php
 create mode 100644 tests/codeception/acceptance/Media/MediaPermissionsCest.php
 delete mode 100755 tests/codeception/acceptance/MediaCest.php
 create mode 100644 tests/codeception/acceptance/SuperFooter/SuperFooterCest.php
 create mode 100644 tests/codeception/functional/Contrib/ExtLinkCest.php"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -126,22 +128,22 @@
...
+        "su-sws/stanford_events": "dev-8.x-1.x",
+        "su-sws/stanford_fields": "dev-8.x-1.x",
+        "su-sws/stanford_image_styles": "dev-8.x-1.x",
+        "su-sws/stanford_news": "dev-8.x-2.x",
+        "su-sws/stanford_notifications": "dev-8.x-1.x",
+        "su-sws/stanford_person": "dev-8.x-1.x",
+        "su-sws/stanford_profile_helper": "dev-8.x-1.x",
+        "su-sws/stanford_ssp": "dev-8.x-1.x",
+        "su-sws/stanford_text_editor": "dev-8.x-1.x",
+        "su-soe/soe_paragraphs": "dev-8.x-1.x"
```
Comment (pookmish) [inline composer.json:146]: gotta fix these

Comment (imonroe) [reply composer.json]: fixed, and squashed back into the previous commit. :)

Comment (pookmish) [reply composer.json]: nice

**Review by pookmish — APPROVED** [review]
Looks good to me once the tests pass

**Comment by imonroe** [comment]
looks like we've never plugged this repo into circleci for some reason?

**Comment by pookmish** [comment]
https://app.circleci.com/pipelines/github/SU-SOE/soe_profile
its there, but not running. CircleCi is having issues right now

**Comment by imonroe** [comment]
Looks like circle started running one of the jobs; it only took 20+ minutes.  🤷‍♂️

**Comment by sherakama** [comment]
https://github.com/Codeception/Codeception/issues/4033

> This may be different case, but I was getting Failed asserting that an array is not empty error for seeLink, when I switched to HTTPS and added auto-redirect from plain HTTP. My tests started to work after I changed WebDriver url from HTTP to HTTPS, however message was very confusing and there may be some bug underneath.

Maybe? Looking at other reasons it fails on circle but not local.

**Comment by pookmish** [comment]
The first failure is `I can see "Site title line"` https://703-193272841-gh.circle-artifacts.com/0/var/www/html/artifacts/acceptance/report.html I'd suggest starting with that

**Comment by sherakama** [comment]
> The first failure is `I can see "Site title line"` https://703-193272841-gh.circle-artifacts.com/0/var/www/html/artifacts/acceptance/report.html I'd suggest starting with that

I see that, but I cannot reproduce it on local. I am going to try to run the whole suite and then run the local footer tests again to see if I can trigger the state that is causing this issue.

**Comment by sherakama** [comment]
I'm at the point where I'll probably get lucky rolling the dice on this one before I figure it out.

**Comment by sherakama** [comment]
Well, on my local, the config import statement in `SubThemeCest.php` is nuking anything after as it re-enables SAML auth.

**Comment by pookmish** [comment]
That config sub theme test should disable saml after though

**Comment by sherakama** [comment]
Circle seems to have an issue with the soe_basic override of line 1. I'm thinking cache or a hook isn't being ran after form save.

**Comment by sherakama** [comment]
PR here: https://github.com/SU-SOE/soe_profile/pull/70

**Pull request closed without merging**